.vib/supabase/vib-publish.json

```
@@ -0,0 +1,84 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/supabase"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/supabase"
+        },
+        "runtime_parameters": "and0OgogIHNlY3JldDogImFiY2RlZmdoaWprbG1ub3BxcnN0dXZ3eHl6MTIzNDU2IgogIGFub25LZXk6ICJleUpoYkdjaU9pSklVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXdvZ0lDQWdJbkp2YkdVaU9pQWlZVzV2YmlJc0NpQWdJQ0FpYVhOeklqb2dJbk4xY0dGaVlYTmxJaXdLSUNBZ0lDSnBZWFFpT2lBeE5qYzFOREF3TkRBd0xBb2dJQ0FnSW1WNGNDSTZJREU0TXpNeE5qWTRNREFLZlEuenR1aUJ6amFWb0ZIbW9salVYV21udURONlFVMldnSklDZXF3eXp5Wk84OCIKICBzZXJ2aWNlS2V5OiAiZXlKaGJHY2lPaUpJVXpJMU5pSXNJblI1Y0NJNklrcFhWQ0o5LmV3b2dJQ0FnSW5KdmJHVWlPaUFpYzJWeWRtbGpaVjl5YjJ4bElpd0tJQ0FnSUNKcGMzTWlPaUFpYzNWd1lXSmhjMlVpTEFvZ0lDQWdJbWxoZENJNklERTJOelUwTURBME1EQXNDaUFnSUNBaVpYaHdJam9nTVRnek16RTJOamd3TUFwOS5xTnNtWHp6NHRHN2VxSlBoMVk1OERidElsSkJhdXdwcXgzOVVGLU13TThrIgoKYXV0aDoKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDIyMjIKbWV0YToKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDIyMjMKcmVhbHRpbWU6CiAgc2VydmljZToKICAgIHBvcnRzOgogICAgICBodHRwOiAyMjI0CnJlc3Q6CiAgc2VydmljZToKICAgIHBvcnRzOgogICAgICBodHRwOiAyMjI1CnN0b3JhZ2U6CiAgc2VydmljZToKICAgIHBvcnRzOgogICAgICBodHRwOiAyMjI2CnN0dWRpbzoKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDIyMjcKCmtvbmc6CiAgc2VydmljZToKICAgIHBvcnRzOgogICAgICBwcm94eUh0dHA6IDgwCg==",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
+            "name": "M4"
+          }
+        }
+      },
+      "actions": [
+        {
+          "action_id": "health-check",
+          "params": {
+            "endpoint": "lb-supabase-kong-http-proxy",
+            "app_protocol": "GENERIC"
+          }
+        },
+        {
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "/.vib/supabase/goss"
+            },
+            "remote": {
+              "workload": "deploy-supabase-kong"
+            },
```

This schema section has changed in the latest CP version and `remote.workload` will be deprecated in the future in favour of `remote.pod.workload`. Although we'll update this catalog-wise soon, we should merge this using the newer schema. 
```suggestion
          "remote": {
            "pod": {
              "workload": "deploy-supabase-kong"
            }
          },
```